packages/react-charts/src/components/ChartBar/ChartBar.tsx

```diff
@@ -440,7 +448,14 @@ export const ChartBar: React.FunctionComponent<ChartBarProps> = ({
   });
 
   // Note: containerComponent is required for theme
-  return <VictoryBar containerComponent={container} theme={theme} {...rest} />;
+  return (
+    <VictoryBar
+      containerComponent={container}
+      theme={theme}
+      dataComponent={ariaLabel && <Bar ariaLabel={({ index }) => ariaLabel[Number(index)]} tabIndex={0} />}
```

Comment:
Curious why `tabIndex` is always set to zero? I believe we should allow devs to set that value -- use a default if necessary.

Reply:
The tab order is odd. Instead of tabbing to each bar in the year 2015, it skips to the next year.

For example, cats 2015, cats 2016, cats 2017, cats 2018, dogs 2015, dogs 2016, etc.

Reply:
> Curious why `tabIndex` is hard coded to zero? I believe we should allow devs to set that value -- use a default if necessary.

If we don't set `tabIndex` here, it will be set to default `-1` in Victory. And in the global style file in Victory, if an element has `-1` as `tabIndex` value, we cannot focus on it. So if we want to focus on an individual bar in our chart, we have to assign it a value.

Reply:
> The tab order is odd. Instead of tabbing to each bar in the year 2015, it skips to the next year.
> 
> For example, cats 2015, cats 2016, cats 2017, cats 2018, dogs 2015, dogs 2016, etc.

Yes, this is the additional issue I mentioned in the PR description. I am also waiting for some suggestions about how to deal with the problem. We can of course set every bar a different `tabIndex` value to control how users Tab through the bars. However, if there are other charts on the same page uses the same `tabIndex`, it will performs like jumping from a chart to the other chart.